### DISCUSSION TITLE
Does this have all functions of pytube?

### DISCUSSION THREAD

**Discussion by dom-ar** [JuanBindez/pytubefix#219] [General]
Since Pytube probably completely stopped working does this project support Pytubes functions like exceptioms and can I extract information like author, title, views, etc if I load the video?

Sorry if the answer is obvious.

**Comment by JuanBindez** [comment]
Hello, your question is great and a lot of people must have the same, and yes Pytubefix was created so that it has all the new features and functions, you just need to change the import from 'from pytube import Youtube' to from pytubefix import Youtube'', We are maintaining compatibility with all pytube features, this project was created precisely to be a continuation of pytube.


See the documentation to see features already existing in pytube and some new ones  http://pytubefix.rtfd.io/

**Reply by dom-ar** [reply]
Great, thank you and good luck maintaining 🫡
